### DISCUSSION TITLE
Show message: Row already exists in DB

### DISCUSSION THREAD

**Discussion by PGLange** [Budibase/budibase#1838] [Help] [answered]
Hi again,
Budibase is addictive! Made me work on a weekend :)
I want to disable a save button and show off a text (Text is done), if a value set is already saved in the db. 
If I can't disable the button, it'd be ok if's css proerties height and width are set to 0....

Is this possible guys? 
You are doing a great job! Thanks for all your help!

**Comment by shogunpurple** [comment, marked as answer]
Thanks @PGLange - thank you! We are thrilled to hear you are enjoying using budibase.

https://github.com/Budibase/budibase/issues/1644 should provide the functionality that you are looking for, which will allow you to  conditionally set properties based on bindings. This will be shipped in the coming week or two.

Thanks!
